news/load_autoloaded.rst

```diff
@@ -0,0 +1,23 @@
+**Added:**
+
+* <news item>
+
+**Changed:**
+
+* <news item>
+
+**Deprecated:**
+
+* <news item>
+
+**Removed:**
+
+* <news item>
+
+**Fixed:**
+
+* load autoloading (but not yet autoloaded) xontribs in tests
```

Comment:
"`update `load_xontrib` pytest fixture to handle auto-loaded xontribs`" would be more appropriate